modules/io/common/src/main/java/org/locationtech/jts/io/geojson/GeoJsonWriter.java

```diff
@@ -88,6 +89,16 @@ public void setEncodeCRS(boolean isEncodeCRS) {
     this.isEncodeCRS  = isEncodeCRS;
   }
   
+  /**
+   * Sets whether the GeoJSON should be output following counter-clocked orientation aka Right Hand Rule defined in RFC7946
```

Comment:
should be "counter-clockwise".

Reply:
Silly mistake, thank you for spotting it.